doc/other-languages.md

```diff
@@ -32,7 +32,8 @@ sourmash_comp_matrix <- as.matrix(sourmash_comp_matrix)
 
 ```
 
-[See the output of plotting and clustering this matrix](_static/ecoli-cmp.html), produced by [this RMarkdown file](_static/ecoli-cmp.Rmd).
+[See the output of plotting and clustering this matrix](https://sourmash.readthedocs.io/en/latest/_static/ecoli-cmp.html),
```

Comment:
This is odd - we can't refer to local static HTML files?

Reply:
Trying to find out, opened an issue in the MyST repo: https://github.com/executablebooks/MyST-Parser/issues/167

Reply:
punted to https://github.com/dib-lab/sourmash/issues/1051